### ISSUE TITLE
Possibility to add sound to notification

### ISSUE THREAD

**Issue by Supereuro** [mirzemehdi/KMPNotifier#34]
Is there a possibility you could enhance the notifier configuration to add (local)sound to notifications?

**Comment by mirzemehdi** [comment]
Yes this can be implemented in future. Or if you want you can also add and create PR for that :)

**Comment by mirzemehdi** [comment]
This is implemented in `1.2.1` version of kmpnotifier @Supereuro.

Add custom notification sound  when initializing the library as below:

**Android:**

@param soundUri  - Custom Notification sound, uri should be converted to String. Default value is null, if null default notification sound will be played. For custom notification sound put notification sound in **_res/raw_** directory. See how it is used in sample app: https://github.com/mirzemehdi/KMPNotifier/pull/61/commits/3708ea8c1e644f0c9bb6819af89b1ddc0468428a

```kotlin
val customNotificationSound =
        Uri.parse(ContentResolver.SCHEME_ANDROID_RESOURCE + "://" + "com.mmk.kmpnotifier.sample" + "/" + R.raw.custom_notification_sound)
    NotifierManager.initialize(
        configuration = NotificationPlatformConfiguration.Android(
            notificationIconResId = R.drawable.ic_launcher_foreground,
            showPushNotification = true,
            notificationChannelData = NotificationPlatformConfiguration.Android.NotificationChannelData(
                soundUri = customNotificationSound.toString()
            )
        )
    )

```

**iOS:**

@param notificationSoundName - Custom notification sound name. Default value is null. If set null, default notification sound will be played. For cusotm notification sound, place the file in the Resources directory of your iOS target. Ensure the file is added to your target's "Build Phases" under "Copy Bundle Resources". See how it is used in sample app: https://github.com/mirzemehdi/KMPNotifier/pull/61/commits/4b2a90dbc5b5eaa09d2a63e8b516308807122173

```kotlin
NotifierManager.initialize(
        NotificationPlatformConfiguration.Ios(
            showPushNotification = true,
            askNotificationPermissionOnStart = true,
            notificationSoundName = "custom_notification_sound.wav"
        )
    )

```

For more details please check: PR https://github.com/mirzemehdi/KMPNotifier/pull/61
